scripts/deploy_testnet_instance.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+"""
+This script is a helper to deploy a Solido instance on the testnet.
+It will create and fund a maintainer account, a multisig owner account, and a multisig account.
```

Comment:
Better to do list:

1. Create and fund a maintainer account
2. Create and fund a multisig owner account
3. Deploy the Solido program
4. Deploy the Multisig program
5. Create a Solido instance
6. ...continue step by step...